src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -413,6 +438,7 @@ protected virtual void Dispose(bool disposing)
                         link.Dispose();
                     }
 
+                    _workerInitTask.SetResult(false);
```

Comment:
Trying to ensure that _workerInitTask dependents don't hang. Potentially should set exception?

Reply:
PR: #4877 handles this